ANIMATED/index.html

```diff
@@ -0,0 +1,134 @@
+                			Here is a caption for this image
+                		</p>
+                	</span>
+                	<img src="http://pop.h-cdn.co/assets/cm/15/05/54cb5d94587ec_-_tallest-buildings-05-1214-lgn.jpg">
+                </a>
+              </div>
+
+              <div class="column one-third">
+                <a href="http://www.morganfeeney.com" target="_blank" title="web design manchester">
+                	<span class="screen">
+                		<p class="title text-gallery">
+                			Here is a caption for this image
+                		</p>
+                	</span>
+                	<img src="http://pop.h-cdn.co/assets/cm/15/05/54cb5d94587ec_-_tallest-buildings-05-1214-lgn.jpg">
+                </a>
+              </div>
+
+              <div class="column one-third">
+                <a href="http://www.morganfeeney.com" target="_blank" title="web design manchester">
+                	<span class="screen">
+                		<p class="title text-gallery">
+                			Here is a caption for this image
+                		</p>
+                	</span>
+                	<img src="http://pop.h-cdn.co/assets/cm/15/05/54cb5d94587ec_-_tallest-buildings-05-1214-lgn.jpg">
+                </a>
+              </div>
+
+              <div class="column one-third">
+                <a href="http://www.morganfeeney.com" target="_blank" title="web design manchester">
+                	<span class="screen">
+                		<p class="title text-gallery">
+                			Here is a caption for this image
+                		</p>
+                	</span>
+                	<img src="http://pop.h-cdn.co/assets/cm/15/05/54cb5d94587ec_-_tallest-buildings-05-1214-lgn.jpg">
+                </a>
+              </div>
+
+              <div class="column one-third">
+                <a href="http://www.morganfeeney.com" target="_blank" title="web design manchester">
+                	<span class="screen">
+                		<p class="title text-gallery">
+                			Here is a caption for this image
+                		</p>
+                	</span>
+                	<img src="http://pop.h-cdn.co/assets/cm/15/05/54cb5d94587ec_-_tallest-buildings-05-1214-lgn.jpg">
+                </a>
+              </div>
+
+              <div class="column one-third">
+                <a href="http://www.morganfeeney.com" target="_blank" title="web design manchester">
+                	<span class="screen">
+                		<p class="title text-gallery">
+                			Here is a caption for this image
+                		</p>
+                	</span>
+                	<img src="http://pop.h-cdn.co/assets/cm/15/05/54cb5d94587ec_-_tallest-buildings-05-1214-lgn.jpg">
+                </a>
+              </div>
+            </div>
+          </div>
+
+        </div>
+
+        <script src="https://code.jquery.com/jquery-{{JQUERY_VERSION}}.min.js" integrity="{{JQUERY_SRI_HASH}}" crossorigin="anonymous"></script>
```

Comment:
What should this `{{JQUERY_VERSION}}` represents? don't you have to change it for something else?